packages/fxa-auth-server/README.md

```diff
@@ -80,8 +80,7 @@ Executing tests using remote databases (MySQL, Redis, Memcached) is possible by
   - REDIS_HOST
   - ACCESS_TOKEN_REDIS_HOST
   - REFRESH_TOKEN_REDIS_HOST
-- Memcached:
-  - MEMCACHE_METRICS_CONTEXT_ADDRESS
+  - METRICS_CONTEXT_ADDRESS
```

Comment:
I'm not sure what this is.  Was it from a search and delete the memcache prefix?

Reply:
Oops yea it was, should be `METRICS_REDIS_HOST`. I'll update and then merge.